### DISCUSSION TITLE
How to fix table columns, sticky header/column?

### DISCUSSION THREAD

**Discussion by shiming1113** [quasarframework/quasar#10617] [Ideas / Proposals] [answered]
I didn't find a good way to fix the columns of the table，As mentioned in the link below : Table with fixed column
[https://element-plus.gitee.io/#/en-US/component/table](url)

**Comment by ontwikkelfabriek** [comment, marked as answer]
@shiming1113 

Perhaps you could explain what is wrong with this:
https://quasar.dev/vue-components/table#example--sticky-column
https://quasar.dev/vue-components/table#example--sticky-header


btw the **link** you post is **not** clickable..

**Reply by shiming1113** [reply]
Thank you. I know how to solve it
